chapter-12-platform-specific-apps/README.md

```diff
@@ -42,10 +42,9 @@ Update your hosts file with the external IP address, similar to the following:
 ### Deploy the Authorization Server
 
 Deploy the authorization server with some preconfigured clients and users.\
-This requires a license file for the particular authorization server we use:
+You will be prompted to download a license file for the particular authorization server we use:
```

Comment:
Only true if there's no `license-override.json`. Document license download and license-override.